src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php

```diff
@@ -117,10 +133,27 @@ protected function getDumpHeader()
             return $this->dumpHeader;
         }
 
-        $line = <<<'EOHTML'
-<script>
+        $line = '<script>
 Sfdump = window.Sfdump || (function (doc) {
 
+var defaultOptions = '.json_encode($this->jsPropertiesDefault).';
```

Comment:
since the actual max depth is provided as argument to the Sfdump function, there is no need to put the default value here (and thus neither no need to handle any merging (`extend`))